deep_speech_2/data_utils/audio.py

```diff
@@ -67,6 +67,54 @@ def from_file(cls, file):
         return cls(samples, sample_rate)
 
     @classmethod
+    def slice_from_file(cls, file, start=None, end=None):
```

Comment:
@reviewers:
No different for `slice_from_file` and `make_silence`. Only re-order the methods.